pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go

```diff
@@ -365,35 +367,22 @@ func manageOpenShiftAPIServerDaemonSet_v311_00_to_latest(
 	operatorConfig *operatorv1.OpenShiftAPIServer,
 	generationStatus []operatorv1.GenerationStatus,
 	forceRollingUpdate bool) (*appsv1.DaemonSet, bool, error) {
-	required := resourceread.ReadDaemonSetV1OrDie(v311_00_assets.MustAsset("v3.11.0/openshift-apiserver/ds.yaml"))
-
-	for i := range required.Spec.Template.Spec.Containers {
-		img := strings.TrimSpace(required.Spec.Template.Spec.Containers[i].Image)
-		switch img {
-		case "${IMAGE}":
-			required.Spec.Template.Spec.Containers[i].Image = imagePullSpec
-		case "${OPERATOR_IMAGE}":
-			required.Spec.Template.Spec.Containers[i].Image = operatorImagePullSpec
-		default:
-			if strings.Contains(img, "$") {
-				return nil, false, fmt.Errorf("invalid image reference %q", img)
-			}
-		}
-	}
-	for i := range required.Spec.Template.Spec.InitContainers {
-		img := strings.TrimSpace(required.Spec.Template.Spec.InitContainers[i].Image)
-		switch img {
-		case "${IMAGE}":
-			required.Spec.Template.Spec.InitContainers[i].Image = imagePullSpec
-		case "${OPERATOR_IMAGE}":
-			required.Spec.Template.Spec.InitContainers[i].Image = operatorImagePullSpec
-		default:
-			if strings.Contains(img, "$") {
-				return nil, false, fmt.Errorf("invalid image reference %q", img)
-			}
-		}
+	tmpl := v311_00_assets.MustAsset("v3.11.0/openshift-apiserver/ds.yaml")
+
+	r := strings.NewReplacer(
```

Comment:
that must be the bit that will replace `encryption-config-${REVISION}`

Reply:
yes